ui/lib/core/addon/helpers/is-active-route.js

```diff
@@ -19,7 +19,7 @@ export default Helper.extend({
     const currentRoute = router.get('currentRouteName');
     let currentURL = router.get('currentURL');
     // if we have any query params we want to discard them
-    currentURL = currentURL.split('?')[0];
+    currentURL = currentURL?.split('?')[0];
```

Comment:
because we're in an engine, this errors out at the curretURL returns a little differently in an engine vs. non-engine.